conda/base/context.py

```diff
@@ -1165,6 +1171,48 @@ def cpu_flags(self):
         info = _get_cpu_info()
         return info["flags"]
 
+    @memoizedproperty
+    @unique_sequence_map(
+        unique_key="backend",
+        property_name="_reporters",
+        allowed_keys={"json", "stdlib"},
+    )
+    def reporters(self) -> Sequence[Mapping[str, str]]:
```

Comment:
I'm curious why we are using the generic types here. Wouldn't it be more explicit/clear for our users if we specified a specific type like a `list[dict[str, str]]`?

Reply:
Agreed, a good rule of thumb is to use abstract inputs and concrete outputs

Reply:
Will make it more specific. I think I will leaving mapping there though because it can theoretically return either a `tuple[dict[str, str]]` or `tuple[frozendict[str,str]]`. Sticking with the generic `Mapping` feels like a more elegant way of defining it.